maintenance-filter-server/src/main/java/org/ccci/maintenance/util/MaintenanceWindowDao.java

```diff
@@ -0,0 +1,18 @@
+package org.ccci.maintenance.util;
+
+import org.ccci.maintenance.MaintenanceWindow;
+import org.joda.time.DateTime;
+
+/**
+ * @author Matt Drees
+ */
+public interface MaintenanceWindowDao
+{
+    MaintenanceWindow getActiveMaintenanceWindow(String filterName, DateTime currentDateTime);
+
+    /** retrieves the window identified by the given id, or null if it does not exist */
+    OwnedMaintenanceWindow getMaintenanceWindowById(String id);
```

Comment:
Seems like more and more of our projects are using Guava.  Maybe it's not a bad idea to add it here, now?